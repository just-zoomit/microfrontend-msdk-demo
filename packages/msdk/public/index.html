<!DOCTYPE html>
<html >
<head>
    <meta charset="utf-8" />
    <!-- <link rel="icon" href="%PUBLIC_URL%/favicon.ico" /> -->
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="theme-color" content="#000000" />
    <meta
      name="description"
      content="Web site created using create-react-app"
    />
    <!-- <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" /> -->
    <!--
      manifest.json provides metadata used when your web app is installed on a
      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
    -->
    <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" /> -->
    <!--
      Notice the use of %PUBLIC_URL% in the tags above.
      It will be replaced with the URL of the `public` folder during the build.
      Only files inside the `public` folder can be referenced from the HTML.

      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
      work correctly both with client-side routing and a non-root public URL.
      Learn how to configure a non-root public URL by running `npm run build`.


    -->
    <title>Zoom Meeting SDK Sample React</title>

    <!-- For Client View -->
    <link type="text/css" rel="stylesheet" href="https://source.zoom.us/2.6.0/css/bootstrap.css" />
    <link type="text/css" rel="stylesheet" href="https://source.zoom.us/2.6.0/css/react-select.css" />

    <!-- Origin Trials to enable Gallery View in Chrome/Edge -->
    <!-- More Info: https://marketplace.zoom.us/docs/sdk/native-sdks/web/advanced/web-isolation -->
    <!-- SharedArrayBuffers in non-isolated pages on Desktop platforms -->
    
    <meta http-equiv="origin-trial" content="AoZJ1s8UztC43SRoB9nM60WPu2h4ctimqRuWgIIzvvcuDOFsVvfS+59jPphoSBIj8gfYgQ8P55EoKJc4HsAhAQgAAABgeyJvcmlnaW4iOiJodHRwOi8vbG9jYWxob3N0OjgwODAiLCJmZWF0dXJlIjoiVW5yZXN0cmljdGVkU2hhcmVkQXJyYXlCdWZmZXIiLCJleHBpcnkiOjE2NjYxMzc1OTl9" />

    
</head>
<body>
    <div id="_msdk-dev-root"></div>
    
</body>
</html>