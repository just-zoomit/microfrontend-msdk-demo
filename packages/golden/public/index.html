<!DOCTYPE html>
<html >
<head>
    
</head>
<body>
    <div id="_golden-dev-root"></div>
    
</body>
</html>